incubator/scylladb/templates/scylla-storageclass.yaml

```diff
@@ -0,0 +1,8 @@
+kind: StorageClass
```

Comment:
Do you need to define a storageclass in the chart? This resource would make the chart required GKE and not portable

Reply:
Are you saying I could completely do away with it?